bundler/bundler_sha1_deprecation_test.go

```diff
@@ -21,12 +21,15 @@ const (
 
 func TestChromeWarning(t *testing.T) {
 	b := newCustomizedBundlerFromFile(t, sha1CA, sha1Intermediate, "")
+
 	s, err := local.NewSignerFromFile(sha1Intermediate, intermediateKey, nil)
+
```

Comment:
nit-pick: prefer no blank line here. Error reporting code is a integrated part of the logic.